volatility3/framework/plugins/windows/pstree.py

```diff
@@ -49,6 +49,12 @@ def get_requirements(cls):
                 description="Process ID to include (with ancestors and descendants, all other processes are excluded)",
                 optional=True,
             ),
+            requirements.BooleanRequirement(
+                name="verbose",
```

Comment:
`verbose` is a little confusing because the main tool already has a `-v` and we have yet to have a flag that doubles in the main tool and a plugin, however it does match with the previous version of volatility and it would need to be explicitly spelled out (`--verbose`).  Lemme know if you think `--details` would work, but it's not a big concern and certainly not a show stopper...

Reply:
Flag no longer needed. Refer to https://github.com/volatilityfoundation/volatility3/pull/1076#discussion_r1449103902